throttling/memory_gcra_test.go

```diff
@@ -22,10 +22,9 @@ func TestMemoryGCRA(t *testing.T) {
 
 		// next request should be allowed after 5 seconds
 		start := time.Now()
-		var allowed bool
 
 		require.Eventually(t, func() bool {
-			allowed, err = l.limit(context.Background(), "key", burst, burst, rate, period)
+			allowed, err := l.limit(context.Background(), "key", burst, burst, rate, period)
 			require.NoError(t, err)
 			return allowed
```

Comment:
This looks like a mistake, we shouldn't have `require`s inside `Eventually` calls. Can you please change it?

```suggestion
			if err != nil {
				return false
			}
			return allowed
```

Reply:
Nevermind, in this case I guess it's fine either way since it's an in-memory implementation it should never return an error, not even at the first try :+1:

Reply:
If it is expected that it should never return an error, should I ignore the error instead of checking it?

Reply:
In this case either logging it or using `require` works fine I guess  :+1: